.golangci.yml

```diff
@@ -70,6 +70,11 @@ issues:
         - gosec
         - perfsprint
         - usestdlibvars
+    # Ignoring gosec G402: TLS MinVersion too low
```

Comment:
This is based on Alex’s PR linked in the description. The same feedback was proposed there and it was fixed, so as soon as I rebase this PR, this will be addressed. Probably after christmas

Reply:
Thanks. Since this PR seems to still be a WIP, I'll move it back to draft. Feel free to mark it as ready for review once you're finished.